packages/ganache-deployer/src/index.ts

```diff
@@ -0,0 +1,48 @@
+import {GanacheServer} from '@statechannels/devtools';
+import fs from 'fs';
+import log from 'loglevel';
+import path from 'path';
+import writeJsonFile from 'write-json-file';
+import {deployContracts} from './deployer';
+
+const GANACHE_CONTRACTS_FILE = 'ganache-network-context.json';
+const GANACHE_CONTRACTS_PATH = path.join(__dirname, '../', GANACHE_CONTRACTS_FILE);
+const GANACHE_CONTRACTS_BACKUP_PATH = path.join(__dirname, '../', `${GANACHE_CONTRACTS_FILE}.bak`);
+
+export async function startGanacheAndDeploy(): Promise<GanacheServer> {
+  // TODO: This should probably merge the existing file instead of replacing it
```

Comment:
You mean it should do this?
1. read the existing file
2. Move it to `$BACKUP_PATH`
3. Merge the new network context with the existing one

Reply:
Yup. I think ideally it should contain the existing test-net deployment info and the ganache info.

Reply:
Should this package and the `ganache-network-context.json` file contain testnet information? From the package and files names, it seems like we should only store ganache addresses.